controller/search_blackbox_test.go

```diff
@@ -1035,77 +1035,40 @@ func (s *searchBlackBoxTest) TestSearchQueryScenarioDriven() {
-		}, spaceOwner.ID)
-	require.Nil(s.T(), err)
-
-	parentWI1, err := wirepo.Create(
-		s.Ctx, *spaceInstance.ID, workitem.SystemTask,
-		map[string]interface{}{
-			workitem.SystemTitle: "Parent WI",
-			workitem.SystemState: workitem.SystemStateResolved,
-		}, spaceOwner.ID)
-	require.Nil(s.T(), err)
-
-	childWI, err := wirepo.Create(
-		s.Ctx, *spaceInstance.ID, workitem.SystemBug,
-		map[string]interface{}{
-			workitem.SystemTitle: "Child WI",
-			workitem.SystemState: workitem.SystemStateResolved,
-		}, spaceOwner.ID)
-	require.Nil(s.T(), err)
-
-	childWI2, err := wirepo.Create(
-		s.Ctx, *spaceInstance.ID, workitem.SystemBug,
-		map[string]interface{}{
-			workitem.SystemTitle: "Child WI",
-			workitem.SystemState: workitem.SystemStateResolved,
-		}, spaceOwner.ID)
-	require.Nil(s.T(), err)
-	// create parent links
-	createPayload := newCreateWorkItemLinkPayload(parentWI0.ID, parentWI1.ID, link.SystemWorkItemLinkTypeParentChildID)
-	_, workItemLink := test.CreateWorkItemLinkCreated(s.T(), s.svc.Context, s.svc, workItemLinkCtrl, createPayload)
-	require.NotNil(s.T(), workItemLink)
-	require.Equal(s.T(), parentWI0.ID, workItemLink.Data.Relationships.Source.Data.ID)
-	require.Equal(s.T(), parentWI1.ID, workItemLink.Data.Relationships.Target.Data.ID)
-	require.Equal(s.T(), link.SystemWorkItemLinkTypeParentChildID, workItemLink.Data.Relationships.LinkType.Data.ID)
-
-	createPayload = newCreateWorkItemLinkPayload(parentWI1.ID, childWI.ID, link.SystemWorkItemLinkTypeParentChildID)
-	_, workItemLink = test.CreateWorkItemLinkCreated(s.T(), s.svc.Context, s.svc, workItemLinkCtrl, createPayload)
-	require.NotNil(s.T(), workItemLink)
-	require.Equal(s.T(), parentWI1.ID, workItemLink.Data.Relationships.Source.Data.ID)
-	require.Equal(s.T(), childWI.ID, workItemLink.Data.Relationships.Target.Data.ID)
-	require.Equal(s.T(), link.SystemWorkItemLinkTypeParentChildID, workItemLink.Data.Relationships.LinkType.Data.ID)
-
-	// in the link below, we keep parentWI0 as parent so that we can verify "included" list is not having duplicates
-	createPayload = newCreateWorkItemLinkPayload(parentWI0.ID, childWI2.ID, link.SystemWorkItemLinkTypeParentChildID)
-	_, workItemLink = test.CreateWorkItemLinkCreated(s.T(), s.svc.Context, s.svc, workItemLinkCtrl, createPayload)
-	require.NotNil(s.T(), workItemLink)
-	require.Equal(s.T(), parentWI0.ID, workItemLink.Data.Relationships.Source.Data.ID)
-	require.Equal(s.T(), childWI2.ID, workItemLink.Data.Relationships.Target.Data.ID)
-	require.Equal(s.T(), link.SystemWorkItemLinkTypeParentChildID, workItemLink.Data.Relationships.LinkType.Data.ID)
+	// keep in mind that TestFixture is going to create 6 items becasue we asked for 3 links
+	// we will ignore extra 2 items and we will use only 4
+	fixtures := tf.NewTestFixture(s.T(), s.DB,
+		tf.WorkItemLinkTypes(1, tf.TopologyNetwork(), func(fxt *tf.TestFixture, idx int) error {
+			fxt.WorkItemLinkTypes[idx].ForwardName = "parent of"
```

Comment:
You should also explicitly specify the topology here.

Reply:
Done in https://github.com/fabric8-services/fabric8-wit/pull/1592/commits/7948645521876b32179c150d65297101014d9d91